test.js

```diff
@@ -5,7 +5,7 @@ var assert = require('assert');
 Object.assign = undefined;
 var objectAssign = require('./');
 
-it('should have the correct length', function () {
+it('should expect at least 2 arguments', function () {
```

Comment:
This is incorrect. It's testing how many arguments are there in the code, just what's passed.